website/content/docs/job-specification/volume_mount.mdx

```diff
@@ -48,6 +49,11 @@ updates to remove a volume that it depends on.
   specify that it is `read_only` on a per mount level using the `read_only`
   option here.
 
+- `propagation_mode` `(string: "")` - Configure [docker bind propagation](https://docs.docker.com/storage/bind-mounts/#configure-bind-propagation),
+   Can be one of `private`、`host-to-task` or `bidirectional` . Empty string and 
+   `private` maps to `rprivate`,  `host-to-task` maps to `rslave`, 
+   and `bidirectional` maps to `rshared`.
+
```

Comment:
There's some formatting glitches here. I was going to just push to your branch but can't because you've got it locked out for maintainer editing. Can you apply this diff please?

```diff
--- a/website/content/docs/job-specification/volume_mount.mdx
+++ b/website/content/docs/job-specification/volume_mount.mdx
@@ -49,10 +49,10 @@ updates to remove a volume that it depends on.
   specify that it is `read_only` on a per mount level using the `read_only`
   option here.

-- `propagation_mode` `(string: "")` - Configure [docker bind propagation](https://docs.docker.com/storage/bind-mounts/#configure-bind-propagation),
-   Can be one of `private`、`host-to-task` or `bidirectional` . Empty string and
-   `private` maps to `rprivate`,  `host-to-task` maps to `rslave`,
-   and `bidirectional` maps to `rshared`.
+- `propagation_mode` `(string: "")` - Configure [docker bind propagation][]. Can
+  be one of `private`、`host-to-task` or `bidirectional`. Empty string and
+  `private` maps to `rprivate`, `host-to-task` maps to `rslave`, and
+  `bidirectional` maps to `rshared`.

 For examples of how to use [HCL2] interpolation for fine-grained control of
 volumes, see [Volume Interpolation].
@@ -60,3 +60,4 @@ volumes, see [Volume Interpolation].
 [volume]: /docs/job-specification/volume 'Nomad volume Job Specification'
 [volume interpolation]: /docs/job-specification/volume#volume-interpolation
 [hcl2]: /docs/job-specification/hcl2
+[docker bind propagation]: https://docs.docker.com/storage/bind-mounts/#configure-bind-propagation
```